### PR TITLE
[WFC] Re-enable form login

### PR DESCRIPTION
**Pull request by rouffj**

### CODE REVIEW

File: modules/sfGuardAuth/lib/BasesfGuardAuthActions.class.php [hunk 1]
```diff
@@ -22,16 +22,84 @@ class BasesfGuardAuthActions extends sfActions
...
+  {
+    throw new sfException('This method is not yet implemented.');
+  }
+
+  private function formLogin($request)
+  {
+     $class = sfConfig::get('app_sf_guard_plugin_signin_form', 'sfGuardFormSignin');
+     $this->form = new $class();
+
+     if ($request->isMethod('post')) {
```
Comment (rpierlot) [inline modules/sfGuardAuth/lib/BasesfGuardAuthActions.class.php]:
``` php
if ($request->isMethod('post') && $this->form->bind($request->getParameter('signin'))->isValid()) {
...

return $this->redirect'('...
}
if ($request->isXmlHttpRequest()) {
     $this->getResponse()->setHeaderOnly(true);
     ...
}
```

**Comment by rouffj** [comment]
@ETSGlobal/ecs-developers feedbacks applied.

**Comment by ch3ric** [comment]
@rouffj be aware this plugin is open sourced and was supposed to be just a git mirror for sfDoctrineGuardPlugin. http://www.symfony-project.org/plugins/sfDoctrineGuardPlugin, with some hot fixes. It should not use any ETS app parameters or maybe we could change this repo to private...

**Comment by rouffj** [comment]
@ch3ric You are right, as soon as we added features related to SSO this plugin is not a mirror of sfDoctrineGuardPlugin anymore.

We can turn this repo private.

**Comment by ch3ric** [comment]
@rouffj OK to add a new private fork with sso features and to keep this public repo without sso features, in case it is used by someone else..

**Pull request closed without merging**